packages/catalog-platformstate-writer/src/consumerServiceV1.ts

```diff
@@ -42,58 +42,45 @@ export async function handleMessageV1(
             dynamoDBClient
           );
 
-          if (
-            existingCatalogEntry &&
-            existingCatalogEntry.version > msg.version
-          ) {
-            // Stops processing if the message is older than the catalog entry
-            return Promise.resolve();
-          } else if (
-            existingCatalogEntry &&
-            existingCatalogEntry.version <= msg.version
-          ) {
-            // suspended->published
-
-            await updateDescriptorStateInPlatformStatesEntry(
-              dynamoDBClient,
-              eserviceDescriptorPK,
-              descriptorStateToItemState(descriptor.state),
-              msg.version
-            );
+          if (existingCatalogEntry) {
+            if (existingCatalogEntry.version > msg.version) {
+              // Stops processing if the message is older than the catalog entry
+              return Promise.resolve();
+            } else {
+              // suspended->published
 
-            // token-generation-states
-            const eserviceId_descriptorId = makeGSIPKEServiceIdDescriptorId({
-              eserviceId,
-              descriptorId: descriptor.id,
-            });
-            await updateDescriptorStateInTokenGenerationStatesTable(
```

Comment:
any reason why this update has been removed?

Reply:
It has just been moved below to avoid code duplication

Reply:
my bad, thank you